.github/workflows/test.yml

```diff
@@ -130,6 +139,11 @@ jobs:
         run: yarn test --filter "${{ inputs.package }}"
         if: ${{ !inputs.skip-tests }}
 
+      - name: Perform CodeQL Analysis
```

Comment:
Should we move it to a separate job and run in parallel?

Reply:
I'm not sure exactly how code scanning works, they seem to recommend running `github/codeql-action/init` first, then running all the usual build steps, and then run `github/codeql-action/analyze` after build is complete. If we need to run the build anyway, then it's probably faster to include it in the existing test.yml than to run the build twice in parallel.

Reply:
Hm.. I probably need to read more on how it actually works.

Reply:
@shumkov Moved to `all-packages.yml` with parallel build. We also get this warning: ` Warning: 1 issue was detected with this workflow: Please specify an on.push hook so that Code Scanning can compare pull requests against the state of the base branch.` and:

![image](https://user-images.githubusercontent.com/32928115/203453148-4a967594-19e6-43a3-ab06-c5b102001b81.png)

Should we add an `on: push` trigger to this workflow? And restrict it to some branches?

Reply:
Yeah, like master and v*-dev ?